package.json

```diff
@@ -16,10 +16,6 @@
     "url": "https://github.com/winjs/react-winjs"
   },
   "license": "MIT",
-  "peerDependencies": {
-    "react": "0.13.x",
-    "winjs": "4.4.x"
-  },
```

Comment:
I'm good either way.

@luisrudge made a comment above that I went ahead and pushed in...

> I think we don't actually need peerDependencies. npm3 won't do anything with it and, for windows users, npm3 is THE way to go. Why don't we go and remove peerDependencies altogether?